test/parallel/test-process-execve-fail.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const { isMainThread, mustCall, skip, isWindows } = require('../common');
+const { throws } = require('assert');
+
+if (!isMainThread) {
+  skip('process.execve is not available in Workers');
+}
+
+if (!isWindows) {
```

Comment:
The docs also mention Android as not being supported. Should that be covered here also?

Reply:
No, @targos was right. I have updated the docs.